cypress/integration/LockedSelector.feature

```diff
@@ -0,0 +1,119 @@
+Feature: In the main page, use the LockedSelector to navigate
+
+  # Main page
+
+  Scenario: Notifying that you need to select org unit and program to get started
+    Given you are in the main page with no selections made
```

Comment:
Till now it might be clear that we are using cypress for the browser manipulation.
However these `.feature` files are part of the cucumber library. Is very human readable and is what the frontend team suggested for us to use.

Now cucumber comes with some keywords. Apart from the `Feature` and `Scenario` we also having the `Given`, `When`, `Then` and the `And` keywords. 

These four are used to describe actions that will take place on the browser itself. There is always a mapping between the text "you are in the main page with no selections made" and a function in the `js` file. You can find this here https://github.com/dhis2/capture-app/pull/828/files#diff-8fe75cae3f49f1e59a01b231043215b7R1